doc/source/user_guide/boundary_conditions.rst

```diff
@@ -10,8 +10,20 @@ and modify cell zone conditions using :ref:`ref_solver_tui_commands`.
 
 Defining boundary conditions
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-The ``define.boundary_counditions.set.velocity_inlet`` TUI command defines velocity
-boundary conditions at inlets.
+The following example shows a comparison between the TUI command and the
+python code for defining velocity boundary conditions at inlets.
```

Comment:
```suggestion
The following example shows a comparison between the TUI command and the
Python code for defining velocity boundary conditions at inlets.
```

Reply:
There are a number of places with that one... :-)